packages/graphiql-react/src/ui/dropdown.tsx

```diff
@@ -1,40 +1,52 @@
 import { ComponentProps, forwardRef, ReactElement } from 'react';
 import { clsx } from 'clsx';
 import { createComponentGroup } from '../utility/component-group';
-import * as DropdownMenu from '@radix-ui/react-dropdown-menu';
+import * as DM from '@radix-ui/react-dropdown-menu';
 
 import './dropdown.css';
 
-const MenuButton = forwardRef<HTMLButtonElement, ComponentProps<'button'>>(
+const Button = forwardRef<HTMLButtonElement, ComponentProps<'button'>>(
   (props, ref) => (
-    <DropdownMenu.Trigger asChild>
+    <DM.Trigger asChild>
       <button
         {...props}
         ref={ref}
         className={clsx('graphiql-un-styled', props.className)}
       />
-    </DropdownMenu.Trigger>
+    </DM.Trigger>
   ),
 );
-MenuButton.displayName = 'MenuButton';
+Button.displayName = 'DropdownMenuButton';
 
-function MenuList({
+function Content({
   children,
   align = 'start',
   sideOffset = 5,
+  className,
   ...props
-}: DropdownMenu.DropdownMenuContentProps): ReactElement {
+}: DM.DropdownMenuContentProps): ReactElement {
   return (
-    <DropdownMenu.Portal>
-      <DropdownMenu.Content align={align} sideOffset={sideOffset} {...props}>
+    <DM.Portal>
+      <DM.Content
+        align={align}
+        sideOffset={sideOffset}
+        className={clsx('graphiql-dropdown-content', className)}
+        {...props}
+      >
         {children}
-      </DropdownMenu.Content>
-    </DropdownMenu.Portal>
+      </DM.Content>
+    </DM.Portal>
   );
 }
 
-export const Menu = createComponentGroup(DropdownMenu.Root, {
```

Comment:
@thomasheyenbrock I renamed Menu to DropdownMenu because

1. for me menu, listbox are confused names
2. we follow radix convention
3. even this file is name dropdown 😅
4. major release is coming so we can do some adjustments :)

Reply:
Sounds good 👍 

Since we're still on `0.x`, I'd even be fine with doing this in a minor. The expectations around `@graphiql/react` always were that it's not yet stable.